### PR TITLE
Add Biba.app 2.14.2503

### PR DESCRIPTION
**Pull request by dysnomian**
Adds a cask for the Biba conferencing tool.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Please don’t indent the values, there should be only one space before them.

In addition, seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/biba.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'biba' do
+  version '2.14.2503'
+  sha256  '8da9fd50f275ad3dd8290da18711094410ea30213741c14b01a2e1b4294f5c1c'
+
+  url 'https://www.biba.com/osx_downloads/dmg'
+  name 'Biba'
+  homepage 'http://www.biba.com'
+  license  :closed
+  app      'Biba.app'
```
Comment (vitorgalvao) [inline Casks/biba.rb]: The should be an empty line on top of this one.

**Comment by dysnomian** [comment]
Amendments made: Whitespace fixes, SHA check removed, version set to latest.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**